### ISSUE TITLE
[IOS] In App Behaviour

### ISSUE THREAD

**Issue by RuanFluid** [OneSignal/react-native-onesignal#270]
Good day,

I've followed the documentation for disabling in app popup notifications, line of code below, but I still get alerts in iOS when receiving push notifications.

self.oneSignal = [[RCTOneSignal alloc] initWithLaunchOptions:launchOptions
                                                       appId:@"[MY_APP_ID]"
                                                       settings:@{kOSSettingsKeyInFocusDisplayOption : @(OSNotificationDisplayTypeNone), kOSSettingsKeyAutoPrompt : @YES}];

Please assist?

**Comment by sachinB94** [comment]
Also facing the same issue with v3.0.4, and react-native 0.42.0

```
self.oneSignal = [[RCTOneSignal alloc] initWithLaunchOptions:launchOptions
                     appId:@"YOUR APP ID"
                     settings:@{kOSSettingsKeyInFocusDisplayOption : @(OSNotificationDisplayTypeNotification), kOSSettingsKeyAutoPrompt : @NO}];
```

**Comment by Nightsd01** [comment]
Fixed in a recent update (3.0.9), you can now use the `setInFocusDisplayType:` from react-native and it should work correctly for both Android and iOS
